chain.go

```diff
@@ -0,0 +1,22 @@
+package netlink
+
+import (
+	"fmt"
+)
+
+// ChainAttrs are the attributes of a Chain
+type Chain struct {
```

Comment:
i wonder if its worth, similar to Filter[1] and Qdisc[2] to abstract this with an interface for consistency.

on the other hand, in contrast to Filter and Qdisc objects we dont have different types of  chains (currently...) so maybe its not worth it. 

just a thought :)

[1] https://github.com/vishvananda/netlink/blob/8e1ce9665a8f8ab9ed94d8830f08781c2951351c/filter.go#L8
[2] https://github.com/vishvananda/netlink/blob/8e1ce9665a8f8ab9ed94d8830f08781c2951351c/qdisc.go#L26

Reply:
ack, makes sense to me.